htmlmaker_postprocessing.js

```diff
@@ -95,6 +106,83 @@ fs.readFile(file, function editContent (err, contents) {
     $(this).after(match[4]);
   });
 
+  // remove Section-Blank-Page sections
+  var sectionBlankPage = $("section.blankpage");
+  sectionBlankPage.remove();
+
+  //// Strip pageBreaks preceding Section starts: Part 1 (of 2):
+  // catch & remove any page break directly preceding Section Starts
+  //  (nothing should be outside of a seciton block, but just in case)
+  var leadingPageBreak = $("section").prev(".PageBreakpb");
```

Comment:
A simpler way to do this is: $(". PageBreakpb + section").prev().remove();
I see what you're doing with the loop (catch a stack of pagebreak paras), however, in that case, let's assume for now that all the extra pagebreaks are intentional. A pagebreak directly preceding a section is redundant, but users might actually want to insert a bunch of blank pages before a section, in which case we should allow them to.